beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -162,6 +162,10 @@ func (s *Service) processFetchedDataRegSync(
 	invalidBlocks := 0
 	blksWithoutParentCount := 0
 	for _, b := range data.bwb {
+		if err := s.cfg.DB.SaveBlobSidecar(ctx, b.Blobs); err != nil {
```

Comment:
The function `processFetchedDataRegSync` doesn't have a single test, this could have been caught, the issue is that it's hard to write tests for receiveBlock.